### PR TITLE
MNTOR-3078 - Update README 

### PR DESCRIPTION
**Pull request by codemist**
# References: 
Jira: MNTOR-3078

# Description

Updating the `readme` for future local env setups

# Checklist (Definition of Done)
- [ ] Localization strings (if needed) have been added.
- [ ] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).
- [ ] I've added or updated the relevant sections in readme and/or code comments
- [ ] I've added a unit test to test for potential regressions of this bug.
- [ ] Product Owner accepted the User Story (demo of functionality completed) or waived the privilege.
- [ ] All acceptance criteria are met.
- [ ] Jira ticket has been updated (if needed) to match changes made during the development process.
- [ ] Jira ticket has been updated (if needed) with suggestions for QA when this PR is deployed to stage.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ the "what" and "why" of data breach alerts.
 
 - [Volta](https://volta.sh/) (installs the correct version of Node and npm)
 - [Postgres](https://www.postgresql.org/) | Note: On a Mac, we recommend downloading the [Postgres.app](https://postgresapp.com/) instead.
+- [Python](https://www.python.org/downloads/) | [With Homebrew](https://docs.brew.sh/Homebrew-and-Python)
```
Comment (pdehaan) [inline README.md:24]: Interesting. I have Python3 installed via homebrew on macOS, but I'm not sure which version is installed by default these days, or which version is included via Xcode utils.

**Comment by flozia** [comment]
I believe we are not using [Redis to preserve local sessions](https://github.com/mozilla/blurts-server?tab=readme-ov-file#preserve-sessions-in-local-development) anymore — so maybe this section can go as well @rhelmer?

**Comment by pdehaan** [comment]
I'll also graciously throw @jespy2, @rafeerahman, and @mozilloid under the bus to review this since they're newest to the project and can probably identify any gaps they individually/collectively in setting the project up locally. ❤️

**Comment by mozilloid** [comment]
@jespy2 James, I remember there was some issue we both encountered when running _pip install ..._, which we solved by using a virtual environment. ([thread](https://mozilla.slack.com/archives/C04BLQVN151/p1715709185472169))

**Comment by jespy2** [comment]
> @jespy2 , I remember there was some issue we both encountered when running _pip install ..._, which we solved by using a virtual environment. ([thread](https://mozilla.slack.com/archives/C04BLQVN151/p1715709185472169)). Do you think there should be something added regarding that?

I seem to remember you mentioning doing this, but I ended up using the standard set up (with significant assistance from others on the team).  Not sure who you'd ask about adding this to the readme.

File: README.md
Comment (jespy2) [inline README.md:1]: Nit:  "needs to be rerun anytime"is grammatically better than "needs re-ran anytime"

image
image

**Review by mozilloid — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (rhelmer) [reply README.md:24]: I think the base system on macOS might be 3 now.

**Comment by rhelmer** [comment]
> > @jespy2 , I remember there was some issue we both encountered when running _pip install ..._, which we solved by using a virtual environment. ([thread](https://mozilla.slack.com/archives/C04BLQVN151/p1715709185472169)). Do you think there should be something added regarding that?
> 
> I seem to remember you mentioning doing this, but I ended up using the standard set up (with significant assistance from others on the team). Not sure who you'd ask about adding this to the readme.

The `npm run build-glean` target (which uses the `@mozilla/glean` node package) is supposed to set up it's own virtual env for their Python dependencies, I'll try reproducing this and file a ticket if it's not working. I think that's the only Python that Monitor currently uses.

**Review by rhelmer — APPROVED** [review]

**Pull request merged**